app/views/work_days/index.html.erb

```diff
@@ -47,12 +50,67 @@
                           method: :delete,
                           data: { confirm: t('.confirm', default: t("helpers.links.confirm", default: 'Are you sure?')) },
                           class: 'btn btn-xs btn-danger' %>
+            <% end %>
           </td>
       </tr>
     <% end %>
   </tbody>
 </table>
 <br>
-
-<%= link_to "New", new_work_day_path, class: 'btn btn-primary' %>
-<%= link_to "Export as PDF", "#", class: 'btn btn-primary' %>
+<% unless @time_sheet.nil? %>
+  <div class="form-group">
```

Comment:
Der Übersichtlichkeit zuliebe könnte man diesen Codeblock in ein partial auslagern. Sowas wie 
```
<% unless @time_sheet.nil? %>
  render 'time_sheet'
<% end %>
```